core/cmd/local_client.go

```diff
@@ -66,8 +69,19 @@ func initLocalSubCmds(client *Client, devMode bool) []cli.Command {
 					Name:  "vrfpassword, vp",
 					Usage: "text file holding the password for the vrf keys; enables Chainlink VRF oracle",
 				},
+				cli.StringSliceFlag{
+					Name:  "config, c",
+					Usage: "TOML configuration file(s) via flag, or raw TOML via env var. If used, legacy env vars must not be set. Multiple files can be used (-c configA.toml -c configB.toml), and they are applied in order with duplicated fields overriding any earlier values. If the 'CL_CONFIG' env var is specified, it is always processed last with the effect of being the final override. [$CL_CONFIG]",
+				},
+				cli.StringFlag{
+					Name:  "secrets, s",
+					Usage: "TOML configuration file for secrets. Must be set if and only if config is set.",
+				},
+			},
+			Usage: "Run the Chainlink node",
+			Before: func(c *cli.Context) error {
+				return initConfigOpts(opts, c)
```

Comment:
What happens if they specify flags at both levels? `chainlink -c configA.toml node start -c configB.toml`
or a mix? `chainlink -c config.toml node start -s secrets.toml`

Reply:
with the latest code,
I manually verified that these all result in the same logs upon startup

chainlink -c <CONFIG> -s <SECRETS> node start -a <API>
chainlink -c <CONFIG>  node start -a <API> -s <SECRETS>
chainlink  -s <SECRETS> node start -a <API> -c <CONFIG>
chainlink node start -a <API> -c <CONFIG> -s <SECRETS>
chainlink -c <CONFIG> -s <SECRETS> node start -a <API> -c <CONFIG> -s <SECRETS>

i modified some of the helper funcs to have better names and added comments that help explain why ^^ works

Reply:
we support multiple config files, so allowing multiple flags like  
chainlink -c c1 -c c2 node start|
so
chainlink -c c1 node start -c c2 
seems  natural to me.

multiple secrets is a bit weird. i can change to error if multiple are given.

Reply:
i think you original instinct is correct - allow (config, secret) in only one command

the bookkeeping a too complex to warrant the effort of tracking config and secret differently.

also, treating them as a group will make it simpler to deprecate the top level instances in the future

now we have:

```
./chainlink -c tmp/config/env.toml -s tmp/config/secrets.toml node start -a tmp/config/api -c tmp/config/env.toml -s tmp/config/secrets.toml 
2023/03/16 16:30:41 Error running app: multiple commands with --config or --secrets flags. only one command may specify these flags. when secrets are used, they must be specific together in the same command
```